tidy3d/plugins/smatrix/component_modelers/modal.py

```diff
@@ -244,7 +246,7 @@ def plot_sim_eps(
             mode_source_0 = self.to_source(port=port_source, mode_index=0)
             plot_sources.append(mode_source_0)
         sim_plot = self.simulation.copy(update={"sources": plot_sources})
-        return sim_plot.plot_eps(x=x, y=y, z=z, ax=ax, **kwargs)
+        return sim_plot.plot_eps(x=x, y=y, z=z, ax=ax, transpose=transpose, **kwargs)
 
```

Comment:
Correct behavior of the `plot_eps()` function was verified (with `transpose=True`) using the [VizSimulation.ipynb](https://github.com/flexcompute/tidy3d-notebooks/blob/develop/VizSimulation.ipynb) notebook example.